rosette/h/BinaryOb.h

```diff
@@ -72,15 +72,15 @@ class ByteVec : public BinaryOb {
         return p[n];
     }
 
-    int numberOfBytes(EMPTY) { return byteCount; }
```

Comment:
I think the official way of declaring this is to use the "void" keyword.  But I'm okay with it either way.

Reply:
http://david.tribble.com/text/cdiffs.htm#C99-empty-parm

Reply:
Since we're c++ only it doesn't matter; a matter of preference.